stingray/simulator/simulator.py

```diff
@@ -311,10 +311,10 @@ def periodogram(self, lc, seg_size=None):
         -------
         ps: numpy.ndarray
             The array of normalized squared absolute values of Fourier
-            amplitudes    
+            amplitudes
 
         """
         if seg_size is None:
             seg_size = self.lc.tseg
 
-        return AveragedPowerspectrum(lc, seg_size).ps
+        return AveragedPowerspectrum(lc, seg_size).power
```

Comment:
Yes it was this one line plus some whitespace corrections by my text editor. I have modified the commit to avoid any merge conflicts when I rebase with the fixed simulator module.